api/src/opentrons/config/advanced_settings.py

```diff
@@ -181,6 +181,11 @@ class Setting(NamedTuple):
         title="Enable robots with the new usb connected rear-panel board.",
         description="This is an Opentrons-internal setting to test new rear-panel.",
     ),
+    SettingDefinition(
+        _id="disableStallDetection",
+        title="Disable stall detection on the OT-3.",
```

Comment:
```suggestion
        title="Disable stall detection on the Flex.",
```
(it's customer facing)